examples/HotelAndTravel/AddHotelAd.php

```diff
@@ -16,7 +16,7 @@
  * limitations under the License.
  */
 
-namespace Google\Ads\GoogleAds\Examples\HotelAds;
+namespace Google\Ads\GoogleAds\Examples\HotelAndTravel;
```

Comment:
WDYT of just `Travel` instead?

Reply:
That's fine too. I just checked and it seems we use Travel to cover both Hotel and Travel now.